vm/src/vm/runners/builtin_runner/signature.rs

```diff
@@ -238,6 +238,10 @@ impl SignatureBuiltinRunner {
                 }))
             }
         }
+        private_inputs.sort_by_key(|input| match input {
+            PrivateInput::Signature(sig) => sig.index,
+            _ => unreachable!(),
```

Comment:
How much work is to add an error handling here?
If it is not possible at least add an error message to the user

Reply:
Added a msg here and also add a test for the signature's `air_public_input` fn.

Reply:
> How much work is to add an error handling here?
> 
> If it is not possible at least add an error message to the user and a line documenting why it is not possible to get into the unreachable

I can add error handling, but I'm not sure how elegant it will be. Lmk if it's important, or if it's okay the way it is now.

Reply:
Also, currently `air_private_input` doesn't return a Result type for any builtin variant. Do we want to `panic` or refactor the code that uses it? Because this case can never happen, I think if we do add an error `panic` is probably okay(?).